index.html

```diff
@@ -4,6 +4,19 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>Simulateur de coût d'embauche</title>
+		<style>
+			body {
+				font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+				line-height: 2.5;
+			}
+
+			/* Customize the widget's style */
+			body .SGMAPembauche {
```

Comment:
Webpack offers an [alternative](https://github.com/webpack/extract-text-webpack-plugin#user-content-extract-text-plugin-for-webpack), building a .css file instead of inline. 
But it requires configuration, and departs from webpack's standards.